package/gluon-logging/luasrc/lib/gluon/upgrade/040-syslog

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/lua
+local site = require('gluon.site')
+local uci = require('simple-uci').cursor()
+
+local system = uci:get_first('system', 'system')
+
+if not site.syslog.ip() then
+	uci:delete_all('system', system, function(entry)
+		return entry['.name']:find('^log_')
```

Comment:
IMHO we should explicitly delete only the 4 entries this package creates.

Reply:
Whats the most elegant way to accomplish that?

Reply:
Maybe like this:
```suggestion
		return entry['.name']:find('^log_(remote|ip|port|proto)')
```

Reply:
`|` does not exist in Lua patterns, and parentheses are only used fore captures, they don't affect the matching itself.

Reply:
`util.contains()` with a list of section names would be an option, but just using `uci:delete` seems like the simpler approach.